packages/react-components/src/index.tsx

```diff
@@ -31,8 +32,10 @@ export { default as Columar } from './Columar';
 export { default as Column } from './Column';
 export { default as ConvictionDropdown } from './ConvictionDropdown';
 export { default as CopyButton } from './CopyButton';
+export { default as CopyToClipboard } from './CopyToClipboard';
```

Comment:
Really not sure about this - prefer a single consistent component for all, so would rather re-use the CopyButton. If it needs replacing, rather do it consistently everywhere.